packages/cli/src/commands/releasecelo/withdraw.test.ts

```diff
@@ -41,7 +41,7 @@ testWithGanache('releasegold:withdraw cmd', (web3: Web3) => {
     await testLocally(Withdraw, ['--contract', contractAddress, '--value', withdrawalAmount])
     const balanceAfter = (await kit.getTotalBalance(beneficiary)).CELO!
     const difference = balanceAfter.minus(balanceBefore)
-    expect(difference).toEqBigNumber(new BigNumber(withdrawalAmount))
+    expect(difference.isEqualTo(new BigNumber(withdrawalAmount))).toBe(true)
```

Comment:
a bit unfortunate as the failure messages no longer will display what the value is

Reply:
can stringify instead I suppose

Reply:
done